doc/web/media/man5/linkcheckerrc.5.html

```diff
@@ -146,30 +146,37 @@ <h2 class="Ss" id="_fB_filtering__fP"><a class="permalink" href="#_fB_filtering_
 <dl class="Bl-tag">
   <dt><b>entry=</b><i>REGEX</i> <i>USER</i> [<i>PASS</i>] (MULTILINE)</dt>
-  <dd>Provide different user/password pairs for different link types. Entries
-      are a triple (URL regex, username, password) or a tuple (URL regex,
-      username), where the entries are separated by whitespace.
+  <dd>Provide individual username/password pairs for different links. In addtion
+      to a single login page specified with <b>loginurl</b> multiple FTP, HTTP
+      (Basic Authentication) and telnet links are supported. Entries are a
+      triple (URL regex, username, password) or a tuple (URL regex, username),
+      where the entries are separated by whitespace.
     <br/>
     The password is optional and if missing it has to be entered at the
       commandline.
     <br/>
-    If the regular expression matches the checked URL, the given user/password
-      pair is used for authentication. The commandline options <b>-u</b> and
-      <b>-p</b> match every link and therefore override the entries given here.
-      The first match wins. At the moment, authentication is used/needed for
-      http[s] and ftp links.
+    If the regular expression matches the checked URL, the given
+      username/password pair is used for authentication. The command line
+      options <b>-u</b> and <b>-p</b> match every link and therefore override
+      the entries given here. The first match wins.
     <br/>
     Command line option: <b>-u</b>, <b>-p</b></dd>
   <dt><b>loginurl=</b><i>URL</i></dt>
-  <dd>A login URL to be visited before checking. Also needs authentication data
-      set for it.</dd>
+  <dd>The URL of a login page to be visited before link checking that uses an
+      HTML form to collect credentials and submit them to the address in its
+      action attribute. The name attributes of the input elements of the form
+      and the values to be submitted need to be available (see <b>entry</b> for
+      an explanation of username and password values).</dd>
   <dt><b>loginuserfield=</b><i>STRING</i></dt>
-  <dd>The name of the user CGI field. Default name is <b>login</b>.</dd>
+  <dd>The name attribute of the username input element. Default:
+    <b>login</b>.</dd>
   <dt><b>loginpasswordfield=</b><i>STRING</i></dt>
-  <dd>The name of the password CGI field. Default name is <b>password</b>.</dd>
+  <dd>The name attribute of the password input element. Default:
+      <b>password</b>.</dd>
   <dt><b>loginextrafields=</b><i>NAME</i><b>:</b><i>VALUE</i> (MULTILINE)</dt>
-  <dd>Optionally any additional CGI name/value pairs. Note that the default
-      values are submitted automatically.</dd>
+  <dd>Optionally the name attributes of any additional input elements and the
+      values to popuplate them with. Note that these are submitted without
+      checking whether matching input elements exist in the HTML form.</dd>
```

Comment:
This is automatically generated and so it would be redundant for me to point out the 'popuplate' typo here?

Reply:
No, it reminds me to regenerate the html.